invokeai/app/services/invocation_cache/invocation_cache_memory.py

```diff
@@ -7,22 +10,29 @@
 from invokeai.app.services.invoker import Invoker
 
 
+@dataclass(order=True)
+class CachedItem:
+    priority: float
```

Comment:
What does this do? I don't see any reference to it in the rest of the implementation - is it related to the dataclass decorator?

Reply:
Ugh, no, good catch. That was for the ordering in my previous implementation. I could have sworn I removed it.